packages/relay/src/lib/clients/cache/redisCache.ts

```diff
@@ -249,10 +249,10 @@ export class RedisCache implements IRedisCacheClient {
    * @returns {Promise<void>} A Promise that resolves when the client is disconnected.
    */
   async disconnect(): Promise<void> {
-    await this.getConnectedClient().then((client) => {
-      client.disconnect();
-      client.unsubscribe();
-    });
+    if (await this.connected) {
```

Comment:
why check this.connected when getConnectedClient actually checks it ?

Reply:
Good point, I removed this redundant check.